src/window/military_menu.c

```diff
@@ -53,14 +58,29 @@ static void draw_foreground(void)
     data.active_buttons = num_legions;
 }
 
+static int click_outside_menu(const mouse *m, int x_offset)
+{
+    return m->left.went_down &&
```

Comment:
Use `went_up` instead of down? That fixes the bug I reported.

Reply:
Yeah, oversight of mine...